.buildkite/bk.integration.pipeline.yml

```diff
@@ -330,19 +338,20 @@ steps:
         agents:
           provider: "gcp"
           machineType: "n1-standard-4"
-          image: "family/platform-ingest-elastic-agent-ubuntu-2404"
+          # image: "family/platform-ingest-elastic-agent-ubuntu-2404"
+          image: "platform-ingest-elastic-agent-ubuntu-2404-1740621659"
           diskSizeGb: 80
         matrix:
           setup:
             variants:
-            - "basic,slim,complete,service,elastic-otel-collector"
-            - "wolfi,slim-wolfi,complete-wolfi,elastic-otel-collector-wolfi"
+              - "basic,slim,complete,service,elastic-otel-collector"
+              - "wolfi,slim-wolfi,complete-wolfi,elastic-otel-collector-wolfi"
             version:
-            - v1.27.16
-            - v1.28.9
-            - v1.29.8
-            - v1.30.8
-            - v1.31.0
+              - v1.27.16
+              - v1.28.9
+              - v1.29.8
+              - v1.30.8
+              - v1.31.0
```

Comment:
Let's only change the images, can you update this?

Reply:
Otherwise, it will cause conflicts to https://github.com/elastic/elastic-agent/pull/7647 unnecessarily

Reply:
First, let's see if the PR changes work. 
This change makes sense because the indentations were wrong. We need to setup pre-commit for yaml to avoid such modifications made by code formatting